### ISSUE TITLE
Diss: cityscpase mIOU result!

### ISSUE THREAD

**Issue by ProWhalen** [CoinCheung/BiSeNet#8]
Thx for author‘s practice !
I have tested  the pretrain model. but i get mIOU=74.6. 
If some one want test, you can download code there: https://github.com/mcordts/cityscapesScripts

So, why your eval code can get 78.45?

Do you consider background class (black color in lable) when training?
I don't think you have done this.

**Comment by CoinCheung** [comment]
Hi, 

Would you be more specific? What is you problem, and how did you reproduce it?

**Comment by REFunction** [comment]
I am also confused. The mIoU is 74.8 in the paper, but you get 78.
Is there any problem with your eval code? Or just improve it by yourself?

**Comment by CoinCheung** [comment]
@REFunction There are many tables about miou in the paper, which table did you get the result of 74.8? I have uploaded my code, if you have suspicions, why not directly point out where the errors are?

**Comment by mengzhibin** [comment]
@REFunction On table 7, there is miou of 78.6, you can check more careful.

**Comment by Aktcob** [comment]
> Thx for author‘s practice !
> I have tested the pretrain model. but i get mIOU=74.6.
> If some one want test, you can download code there: https://github.com/mcordts/cityscapesScripts
> 
> So, why your eval code can get 78.45?
> 
> Do you consider background class (black color in lable) when training?
> I don't think you have done this.

if test with single scale, mIoU=74.6 is not a bad result. And the Eval in this repo can easily get much better performance.

**Comment by stuhailiu** [comment]
@CoinCheung Thx for your practice !  I just changed InPlaceABNSync to nn.BatchNorm2d so that evaluate.py can run on a single GPU, but the miou I got is only 0.002845. Could you tell me how to solve it ?

**Comment by XUYUNYUN666** [comment]
any one who has trained the network on your own dataset?

**Comment by CoinCheung** [comment]
No more information on this topic ? I am closing this.
